### DISCUSSION TITLE
A way to see sweeper hits without having to go into the game

### DISCUSSION THREAD

**Discussion by ReinSonozaki** [c0nD/FluffyBOT#57] [Ideas]
Maybe a way to like /sweeper_list and it will show how many hits each sweeper has done or has? 

eg: 
Sweeper 1 
Aod: 5/9 
Tla: 2/9

**Comment by c0nD** [comment]
Related to issue #7 

Def. something that should be considered for implementation.

Most likely would be a role ID where users of that role would have a counter() for each boss.

@OneStone2 -- thoughts?
